Makefile

```diff
@@ -51,13 +51,24 @@ format: $(GO)
 	$(GO) fmt ./pkg/... ./cmd/...
 	$(GO) vet ./pkg/... ./cmd/...
 
-test: $(GO)
+test-build-host-local-plugin:
```

Comment:
```suggestion
build-host-local-plugin:
```

Reply:
Removing the prefix since otherwise it conflicts with the test-* target.

Reply:
Fixed